pkg/kp/pcstore/consul_store_test.go

```diff
@@ -231,6 +231,57 @@ func TestDelete(t *testing.T) {
 	}
 }
 
+func TestList(t *testing.T) {
+	store := consulStoreWithFakeKV()
+	podID := types.PodID("pod_id")
+	az := fields.AvailabilityZone("us-west")
+	clusterName := fields.ClusterName("cluster_name")
+
+	selector := klabels.Everything().
```

Comment:
I don't actually care about the content of the selector here so i'll make it `klabels.Everything()`